### PR TITLE
Font size

### LINKED ISSUE #402: Update text styling on page 2 of new survey
## Summary

- [ ] Kind of open ended, but we want to make our text more appealing to the user, as this is the core part of our app

## Motivation

Making the survey look as good as it possibly can

## Describe alternatives you've considered

- Leaving this as is

### LINKED ISSUE #418: [BLOCKED BY #427] Stylize survey page 2 to look better on desktop
## Summary

- [ ] there is no wireframe for this and this can be more greenfield (so UI-types encouraged!), but we want the SurveyPage2 component to look "better" on desktop, as currently it's usable but not very inviting. 
- [ ] uses coloring, styling similar to other components 

## Motivation

While this is a mobile-first app, we still want our application to be presentable on desktop

## Describe alternatives you've considered

Leaving it ugly

### LINKED ISSUE #422: [BLOCKED BY #427] Survey page 2 is all messed up with scrolling on mobile (iPhone X)
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in QA
    * Followed all applicable steps in the TROUBLESHOOTING guide: TODO
    * Checked for relevant discussions on Discord: <https://discord.gg/pPERUuv>
    * Checked that your issue isn't already filed: <https://github.com/COVID-19-electronic-health-system/Corona-tracker/issues>

### Description

When taking the survey on mobile, page 2 is a nightmare to use due to scrolling issues within each grid item. Not represented well by the gif, but you're able to scroll within each row

### Steps to Reproduce

1. Visit app on mobile http://coronatrackerv1.s3-website-us-east-1.amazonaws.com/
2. Take survey

**Expected behavior:**

The survey is appealing to the user

**Actual behavior:**

The scrolling issues make the app almost unusable 

**Reproduces how often:**

100%

### Versions

3.0.2

### LINKED ISSUE #427: Setup global fonts and screen breaking points
## May block issues:
#422 #418 #402

## Plan
- [ ] Setup the fonts globally to be used with the help of @BrianHHough 
- [ ] Setup breaking points also globally using Material-UI
- [ ] Update the styling guide to how to use 
### Motivation:
- start using the fonts to better reflect the wireframes
- Make a united setup for everyone which make it easier to debug and prevent conflict between components as everything following the same setup

## Alternative
leave it as it is so it is added in every component spritely

### PR DESCRIPTION
**Pull request by AdhamAH**
## IMPORTANT: Please do not create a Pull Request without creating an issue first.

**Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.**

**Do not change the following line aside from replacing YOUR-ISSUE-HERE with the issue this PR fixes** 

fixes #427 fixes #422 fixes #418

### Test plan (required)

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

### Larg screens
Screenshot 2020-04-08 at 22 51 07

### Scrolling and buttons
last

### Final Checklist

- [x] Have you bumped the version in `package.json`?
    - Second decimal for major change, third decimal for minor change. This can go past 10 i.e. 1.0.9 !=> 1.1.0, 1.0.9 => 1.0.10
    - [Read here for more info](https://semver.org/)
- [ ] Have you added any new tests necessary?
- [x] Is your PR rebased off the most current master?
- [x] Have you squashed all commits, or if you will merge will you squash all commits?
- [x] Did you use yarn, not npm?
- [x] Did you use Material-UI wherever possible?
- [x] Did you format according to Prettier?
- [x] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

**Comment by AdhamAH** [comment]
I made a mistake I think it doesn't close #402 @SomeMoosery

File: client/src/components/survey/SurveyPage1.jsx [hunk 1]
```diff
@@ -23,8 +23,8 @@ const useStyles = makeStyles(() => ({
     justifyContent: 'center',
     marginLeft: 'auto',
     marginRight: 'auto',
-    maxWidth: '992px',
```
Comment (acthelemann) [inline client/src/components/survey/SurveyPage1.jsx:26]: See my comment.

File: client/src/components/survey/SurveyPage1.jsx [hunk 2]
```diff
@@ -161,13 +161,13 @@ const SurveyPage1 = props => {
             <b>Let&apos;s answer a few questions!</b>
           </Typography>
         </Grid>
-        <Grid item xs={12}>
-          <Typography>
+        <Grid item xs={12} xl={4}>
```
Comment (acthelemann) [inline client/src/components/survey/SurveyPage1.jsx:164]: See my comment.

File: client/src/components/survey/SurveyPage1.jsx [hunk 3]
```diff
@@ -161,13 +161,13 @@ const SurveyPage1 = props => {
...
+          <Typography variant="body1">
             Be sure to answer truthfully and honestly so that your health record will be accurate and helpful
           </Typography>
         </Grid>
-        <Grid item xs={12}>
-          <Typography id="discrete-slider" gutterBottom>
+        <Grid item xs={12} xl={4} className={classes.gridItem}>
```
Comment (acthelemann) [inline client/src/components/survey/SurveyPage1.jsx:169]: See my comment.

File: client/src/components/survey/SurveyPage1.jsx (hunk 1 above)
Comment (AdhamAH) [reply client/src/components/survey/SurveyPage1.jsx:26]: I agree! it looks weird.
I didn't look into it  😅 I was trying to make this up and running so from now on everything follow the same sizing and fonts
If you saw discord we were talking about finish what we have now and start work on making such improvement

**Comment by AdhamAH** [comment]
@crispinamuriel remember that so we can work on it later, thanks

**Pull request merged**